design-system/chip.tsx

```diff
@@ -73,9 +76,9 @@ const $chip: ThemedStyle<ViewStyle> = ({
   borderWidth: borderWidth.sm,
   borderColor: colors.border.subtle,
   paddingVertical: spacing.xxs,
-  paddingHorizontal: spacing.xs,
-  minHeight: 36,
-  //   ...debugBorder("orange"),
+  paddingHorizontal: spacing.sm,
+  minHeight: 32,
```

Comment:
when we have a value like that i like putting a comment like " // From Figma" or something like that so other can understand why we didn't use the theme

Reply:
slash also get on Andrew to consolidate if there is spacing that isn't declared that he uses 

the sooner we get figma aligned with our sytles the less we have to deal with this crap